compiler/crates/relay-config/src/project_config.rs

```diff
@@ -383,6 +383,21 @@ impl ProjectConfig {
         self.create_path_for_artifact(source_file, filename)
     }
 
+    pub fn path_for_language_specific_artifact(
```

Comment:
why do we need a language specific `path_for_artifact`?

Reply:
`path_for_artifact` adds `.graphql.{js|ts}` as an extension. In the previous PR and from what I've read, the $parameters files have been named `<OperationName>$parameters.{js|ts}`, without the `.graphql`.

If desired I can of course re-use the existing method, but the artifacts would be named differently then 😊

Reply:
Can we also use this method to generate path `for *.graphql.js` artifacts?

Reply:
So here:

https://github.com/facebook/relay/pull/4515/files#diff-c4a1061d9a7a07501974396d27729b0a9b4bac18dd4729ee2982bbb6ff412a42R379-R385

We would call to `self.path_for_language_specific_artifact(format!("{}.graphql.ts", artifact_name)))`